library-app/src/helpers/roles.tsx

```diff
@@ -0,0 +1,19 @@
+import { Jwt, roleKey, UserRole } from '../models/jwt.model';
+// eslint-disable-next-line camelcase
+import jwt_decode from 'jwt-decode';
+
+export const currentUserAdmin = (token: string) => {
```

Comment:
Necemo token da prosledjujemo ovom, nego neka ovaj servis pozove token servis koji samo procita token is local storage-a

Reply:
jasno jasno, ima vise smisla :D